### DISCUSSION TITLE
have a difficultly in detrmining parameter

### DISCUSSION THREAD

**Discussion by neperchh** [neperfepx/neper#768] [Q&A]
Dear teacher, I had a EBSD date, but when I construct a model by using neper ,how to determine the parameter of sphericity. Besides diameter and circularity , can diameter and other parameters be used. Thanks very much.

**Comment by rquey** [comment]
The sphericity (or circularity) is defined in the [doc](https://neper.info/doc/neper_t.html#sphericity-circularity):
> Terms sphericity and circularity apply to 3D and 2D, respectively, but can be used interchangeably. The sphericity of a polyhedron corresponds to the ratio of the surface area of the sphere of equivalent volume to the surface area of the polyhedron. Similarly, the circularity of a polygon corresponds to the ratio of the perimeter of the circle of equivalent surface area to the perimeter of the polygon.
However, in Neper, they cannot be computed from a tesr (EBSD map).  So, you'll have to do it externally, or "guess" a value in Neper and see how it compares with the experiment.

The morphology parameters are listed in [`-morpho`](https://neper.info/doc/neper_t.html#cmdoption-morpho), and are indeed the diameter (size parameter) and sphericity/circularity (shape parameter).
